dspace-server-webapp/src/test/java/org/dspace/app/rest/BitstreamRestControllerIT.java

```diff
@@ -349,6 +351,170 @@ public void testEmbargoedBitstream() throws Exception {
+                   .withAdminGroup(adminParentCommunity)
+                   .build();
+
+        Community child1 = CommunityBuilder.createSubCommunity(context, parentCommunity)
+                   .withName("Sub Community")
+                   .build();
+
+        EPerson adminCollection1 = EPersonBuilder.createEPerson(context)
+                .withEmail("adminCollection1@mail.com")
+                .withPassword("qwerty03")
+                .build();
+        Collection col1 = CollectionBuilder.createCollection(context, child1)
+                   .withName("Collection 1")
+                   .withAdminGroup(adminCollection1)
+                   .build();
+
+        EPerson adminCollection2 = EPersonBuilder.createEPerson(context)
+                .withEmail("adminCol2@mail.com")
+                .withPassword("qwerty01")
+                .build();
+        Collection col2 = CollectionBuilder.createCollection(context, child1)
+                   .withName("Collection 2")
+                   .withAdminGroup(adminCollection2)
+                   .build();
+
+
+        String bitstreamContent = "Embargoed!";
+        Bitstream bitstream = null;
+        try (InputStream is = IOUtils.toInputStream(bitstreamContent, CharEncoding.UTF_8)) {
+
+            Item item = ItemBuilder.createItem(context, col1)
+                    .withTitle("Test")
+                    .withIssueDate("2018-10-18")
+                    .withAuthor("Smith, Donald")
+                    .withSubject("ExtraEntry")
+                    .build();
+
+            bitstream = BitstreamBuilder.createBitstream(context, item, is)
+                    .withName("Bitstream")
+                    .withDescription("Description")
+                    .withMimeType("text/plain")
+                    .withEmbargoPeriod("2 week")
+                    .build();
+        }
+        context.restoreAuthSystemState();
+
+        // parent community's admin user is allowed access to embargoed item
+        String tokenAdminParentCommunity = getAuthToken(adminParentCommunity.getEmail(), "qwerty02");
+        getClient(tokenAdminParentCommunity).perform(get("/api/core/bitstreams/" + bitstream.getID() + "/content"))
+                   .andExpect(status().isOk());
+
+        // collection1's admin user is allowed access to embargoed item
+        String tokenAdminCollection1 = getAuthToken(adminCollection1.getEmail(), "qwerty03");
+        getClient(tokenAdminCollection1).perform(get("/api/core/bitstreams/" + bitstream.getID() + "/content"))
+                   .andExpect(status().isOk());
+
+        checkNumberOfStatsRecords(bitstream, 2);
+
+        // admin of second collection is NOT allowed access to embargoed item  of first collection
+        String tokenAdminCollection2 = getAuthToken(adminCollection2.getEmail(), "qwerty01");
+        getClient(tokenAdminCollection2).perform(get("/api/core/bitstreams/" + bitstream.getID() + "/content"))
+                   .andExpect(status().isForbidden());
```

Comment:
Please add a similar Forbidden test for a Community Admin of a (new) Community which is not a parent Community of collection1.